test/helpers/start-app.js

```diff
@@ -22,5 +27,18 @@ module.exports = async function startApp(appPath) {
     });
   });
 
+  emberServe.shutdown = async function() {
+    this.kill();
+    try {
+      await this;
+    } catch (e) {
+      // Process is allowed to exit with a non zero exit status code.
+    }
+    if (!this.killed) {
+      throw new Error(`the process ${this.pid} wasn't killed.`);
```

Comment:
This may not be needed, this `await this` should not resolve until the process exits.  If we are unsure, we can leave it in. Seems unlikely to be a bad thing.